packages/flutter_driver/lib/src/driver.dart

```diff
@@ -211,15 +211,21 @@ class FlutterDriver {
     return await _sendCommand(new Scroll(finder, dx, dy, duration, frequency)).then((Map<String, dynamic> _) => null);
   }
 
+  /// Tell the driver to ensure that the widget located by [finder] has been
+  /// scrolled completely into view. See [Scrollable.ensureVisible].
```

Comment:
`[Scrollable.ensureVisible]` is not linkable from this package as this code does not import `package:flutter`. You might need to provide an absolute link to [Scrollable.ensureVisible](http://docs.flutter.io/flutter/widgets/Scrollable/ensureVisible.html) on the API docs site for the link to work.